_posts/en/newsletters/2018-07-24-newsletter.md

```diff
@@ -0,0 +1,169 @@
+  performing whatever actions they desire, such as using the
+  [`fundrawtransaction`][rpc fundrawtransaction] RPC.
+
+- [#12196][]: New `scantxoutset` RPC method that allows searching the
+  set of spendable bitcoins (UTXOs) for those matching an address,
+  public key, private key, or HD keypath.  The main expected use for
+  this is "funds sweeping" where transactions matching an old wallet are
+  found and transferred to a new wallet.  Although this RPC will almost
+  certainly be included in Bitcoin Core 0.17, it will likely be marked
+  as experimental so that its API can be freely changed in subsequent
+  releases.  As noted in the News section, this API is likely to be
+  updated to support output script descriptors, which is planned to
+  happen before 0.17.
+
+- [#13604][]: Bitcoin-Qt is now built by default in addition to bitcoind
+  on 32-bit ARM systems, and should be distributed by default with the
+  other binaries for that system from BitcoinCore.org for future
+  releases.  Bitcoin-Qt with 64-bit ARM is not yet supported by default.
+
+- [#13298][]: The node now sends all announcements ([invs][inv]) for
+  new transactions to all of its incoming peers at the same time, after
+  a random delay.  Previously, Satoshi Nakamoto [added a feature][rand
+  delay] to Bitcoin (the software) that waited for a different random
+  delay for each peer before sending an announcement so that a
+  transaction would propagate around the network somewhat unpredictably,
+  preventing spy nodes from being able to assume that the first peer
+  they received a transaction from was likely the peer that created it.
+
+    However, later investigators realized that someone operating
+    multiple spy nodes could make multiple connections to each node to
+    increase their chances of being the first to receive a given
+    transaction, allowing the spy to again guess which node create the
+    transaction.  This merge improves the situation by preventing a spy
+    making multiple connections from receiving any more information than
+    a spy with one connection.  Outgoing connections (which are selected
+    by the node itself using certain rules) continue to use the old
+    behavior so that transactions continue to propagate unpredictably.
+
+    This change might increase transaction propagation delay slightly,
+    although developers commenting on the PR think the effect will be
+    minimal.  It may also cause bandwidth usage to be less evenly
+    distributed over time.   However, it could (in theory) end up
+    reducing the number of incoming connections to upgraded nodes, if spy
+    nodes no longer find making multiple connections to be useful,
+    reducing overall wasted bandwidth.
+
+- [#13652][]: The [`abandontransaction`][rpc abandontransaction] RPC has
+  been fixed to abandon all descendant transactions, not just children.
+
+**Coming attractions:** next week's newsletter will feature a field
```

Comment:
Great teaser! I think this deserves its own section, so changed this to be an h2